examples/pxScene2d/src/pxText.cpp

```diff
@@ -115,6 +115,34 @@ void pxText::resourceReady(rtString readyResolution)
   }     
 }
 
+void pxText::update(double t)
+{
```

Comment:
Can you please move the FBO creation and rendering from the update() method to draw()?